recipes/newrelic/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.68.0.50" %}
+
+package:
+  name: newrelic
+  version: {{ version }}
+
+source:
+  fn: newrelic-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/newrelic/newrelic-{{ version }}.tar.gz
+  sha256: 3ac02183910fe41ab75485d05474164890b993d082dca5847b0e6d24cf166f89
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - glib  # [unix]
```

Comment:
It actually loads pkg_resources in a number of places at run time to
discover extensions... is there some other package that offers
pkg_resources?

On Thu, Jul 21, 2016 at 9:32 AM jakirkham <notifications@github.com> wrote:

> In recipes/newrelic/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/763#discussion_r71704872>
> :
>
> > +  name: newrelic
> > +  version: {{ version }}
> > +
> > +source:
> > +  fn: newrelic-{{ version }}.tar.gz
> > +  url: https://pypi.io/packages/source/n/newrelic/newrelic-{{ version }}.tar.gz
> > +  sha256: 3ac02183910fe41ab75485d05474164890b993d082dca5847b0e6d24cf16
>
> 6f89
> > +
> > +build:
> > +  number: 0
> > +  script:
> > +    - python setup.py install --single-version-externally-managed --record=record.txt
> > +
> > +requirements:
> > +  build:
> > +    - glib  # [unix]
>
> Could we please try adding the entry points to the meta.yaml? Maybe we
> can get around that setuptools dependency.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/763/files/e8511a5c93c7e8cf45774731dd689ee8fc663f90#r71704872>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACxRKGJZmw3wGuDQPhuj0708WZ5A3F8ks5qX3TfgaJpZM4Iz2QG>
> .
>